lib/src/main/java/com/auth0/spring/security/api/JwtAuthenticationEntryPoint.java

```diff
@@ -9,8 +10,24 @@
 import java.io.IOException;
 
 public class JwtAuthenticationEntryPoint implements AuthenticationEntryPoint {
+
+    final String audience;
+    final String issuer;
+
+    public JwtAuthenticationEntryPoint(String audience, String issuer) {
```

Comment:
We will need to provide a default no-arg constructor, to ensure that existing usages of this class aren't broken with this change.